src/main/java/uk/gov/companieshouse/itemhandler/mapper/OrderDataToCertifiedCopyOrderConfirmationMapper.java

```diff
@@ -58,13 +63,24 @@ default void mapCertifiedCopyItems(final OrderData order,
         List<CertifiedDocument> certifiedDocuments = new ArrayList<>();
         IntStream.range(0, filingHistoryDocuments.size()).forEach(i -> {
             CertifiedDocument certifiedDocument = new CertifiedDocument();
-            certifiedDocument.setDateFiled(filingHistoryDocuments.get(i).getFilingHistoryDate());
+            certifiedDocument.setDateFiled(reformatDateFiled(filingHistoryDocuments.get(i).getFilingHistoryDate()));
             certifiedDocument.setType(filingHistoryDocuments.get(i).getFilingHistoryType());
             certifiedDocument.setDescription(filingHistoryDocuments.get(i).getFilingHistoryDescription());
-            certifiedDocument.setFee("£" + itemCosts.get(i).getCalculatedCost());
+            certifiedDocument.setFee(itemCosts.get(i).getCalculatedCost());
             certifiedDocuments.add(certifiedDocument);
         });
 
         return certifiedDocuments;
     }
+
+    /**
+     * Reformats a date filed string such as "2009-08-23" as "23 Aug 2009".
+     * @param dateFiled the date filed as reported from the filing history
+     * @return the same date rendered for display purposes
+     */
+    @Named("reformatDateFiled")
+    default String reformatDateFiled(final String dateFiled) {
```

Comment:
The MapperUtil has some utility methods. Should this live in there as well?

Reply:
It's only used in one place, and it's currently located close to its point of use.